apps/marketplace/components/Reports/ReportItem.js

```diff
@@ -0,0 +1,64 @@
+// will swap this with <img>
+/* eslint-disable react/no-danger  */
+/* eslint-disable import/no-dynamic-require */
```

Comment:
is this still required?

Reply:
I will swap `dangerouslyset` for an `img` but for now eslint complains